src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -456,4 +456,15 @@ public static File regularReadableUserFile(final File file) {
             return file;
         }
     }
+
+    /**
+     * Invert logic if specified
+     *
+     * @param logic boolean logical operation value
+     * @param invert whether to invert logic
+     * @return invert logic if invert flag is true, otherwise leave the logic
+     */
+    public static boolean invertLogic(final boolean logic, final boolean invert){
+        return invert ? !logic : logic;
```

Comment:
This `invertLogic()` method is an abomination... you've replaced simple negation with a method that takes multiple boolean parameters. Please either refactor away or move to a private helper method inside `VariantFiltration` so that no one else will ever use it to confuse future generations of readers.